MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionViewQtGUI.cpp

```diff
@@ -1405,6 +1405,40 @@ std::string EnggDiffractionViewQtGUI::getFittingRunNo() const {
   return m_uiTabFitting.lineEdit_pushButton_run_num->text().toStdString();
 }
 
+void MantidQt::CustomInterfaces::EnggDiffractionViewQtGUI::
+    clearFittingComboBox() const {
+  m_uiTabFitting.comboBox_bank->clear();
+}
+
+void MantidQt::CustomInterfaces::EnggDiffractionViewQtGUI::
+    enableFittingComboBox(bool enable) const {
+  m_uiTabFitting.comboBox_bank->setEnabled(enable);
+}
+
+void MantidQt::CustomInterfaces::EnggDiffractionViewQtGUI::
+    clearFittingListWidget() const {
+  m_uiTabFitting.listWidget_fitting_run_num->clear();
+}
+
+void MantidQt::CustomInterfaces::EnggDiffractionViewQtGUI::
+    enableFittingListWidget(bool enable) const {
+  m_uiTabFitting.listWidget_fitting_run_num->setEnabled(enable);
+}
+
+int MantidQt::CustomInterfaces::EnggDiffractionViewQtGUI::
+    getFittingListWidgetCurrentRow() const {
+  return m_uiTabFitting.listWidget_fitting_run_num->currentRow();
+}
+
+void MantidQt::CustomInterfaces::EnggDiffractionViewQtGUI::
+    setFittingListWidgetCurrentRow(int idx) const {
+  m_uiTabFitting.listWidget_fitting_run_num->setCurrentRow(idx);
+}
+
+int EnggDiffractionViewQtGUI::getFittingComboIdx(std::string bank) const {
+  return m_uiTabFitting.comboBox_bank->findText(QString::fromStdString(bank));
+}
+
```

Comment:
This block of methods is a perfect example of what the ideal view code should look like. All are pasively passing around values, to/from the widgets/user.